SonarQube.Client/Api/V5_50/GetRulesRequest.cs

```diff
@@ -99,6 +101,9 @@ private class RuleResponse
 
             [JsonProperty("repo")]
             public string RepositoryKey { get; set; }
+
+            [JsonProperty("type")]
+            public string Type { get; set; }
```

Comment:
Fortunately, there is only one version of `GetRulesRequest` i.e. the API hasn't changed since this version was created.
Even better, I didn't need to change the request to include additional data since the returned json already contained the type information.
All I needed to do was deserialize the json and convert it to an enum.